handlers/issue/title/label/handler.go

```diff
@@ -0,0 +1,130 @@
+package label
+
+import (
+	`context`
+	`strings`
+
+	`github.com/google/go-github/v35/github`
+	`github.com/sirupsen/logrus`
+
+	`github.com/erda-project/erda-bot/gh`
+	`github.com/erda-project/erda-bot/handlers`
+	`github.com/erda-project/erda-bot/handlers/issue/comment`
+)
+
+var semanticLabelMap = map[string]string{
+	"feat":     "feature",
+	"fix":      "bugfix",
+	"refactor": "refactor",
+	"docs":     "",
+	"style":    "",
+	"perf":     "",
+	"test":     "",
+	"build":    "",
+	"ci":       "",
+	"chore":    "",
+	"revert":   "",
+}
+
+type prTitleLabelHandler struct{ comment.IssueCommentHandler }
+
+func NewPrTitleLabelHandler(nexts ...handlers.Handler) *prTitleLabelHandler {
+	return &prTitleLabelHandler{*comment.NewIssueCommentHandler(nexts...)}
+}
+
+func (h *prTitleLabelHandler) Execute(ctx context.Context, req *handlers.Request) {
+	e := req.Event.(github.IssueCommentEvent)
+	labelsNew, err := findNewLabels(e)
+	if err != nil {
+		logrus.Errorf("failed to findNewLabels, err: %v", err)
+		return
+	}
+	if len(labelsNew) == 0 {
+		h.DoNexts(ctx, req)
+	}
+	labelsEffect, err := findEffectLabels(e, labelsNew)
```

Comment:
no need find twice.
just find required labels and add idempotently.

Reply:
done